deploy/helm/sumologic/values.yaml

```diff
@@ -242,7 +242,16 @@ sumologic:
     collector:
       otelcol:
         enabled: true
-
+      otelcloudwatch:
+        enabled: false
+        roleArn: ""
+        awscloudwatch:
+          region: us-east-2
+          logs:
+            poll_interval: 1m
+            groups:
+              autodiscover:
+                limit: 10
```

Comment:
Do we need to expose full configuration to customers?
I would rather keep:

```
sumologic:
  logs:
    cloudwatch:
      collector:
        enabled: true
      roleArn: ""
      region: ""
      streams:
        autodiscover: true/false
        names: []
        prefixes: []
      ...
```

and `merge` and `override` in addition to that

Also my question is are we able to autodetect region, roleArn, names and/or prefixes of the cluster we are installed in?

Reply:
Group name is determined by fluent-bit config.
Stream names seem to have format `<FLUENT_BIT_PREFIX>-kube.var.log.containers.<POD_NAME>_<NAMESPACE_NAME>_<CONTAINER_NAME>-<LOGFILE_NAME>`
I'm not sure about the rest.

If we are to change the receiver config as you suggest, then we should also mention in the docs that `merge` and `override` can be used.

Reply:
Shouldn't we actually generate `fluent-bit` config with our helm chart for the most basic cases?

Reply:
> If we are to change the receiver config as you suggest, then we should also mention in the docs that merge and override can be used.

We will have at some point, but we just get out from exposing full configuration for other components so we shouldn't introduce the same approach again and again :sweat_smile:

Reply:
> Shouldn't we actually generate fluent-bit config with our helm chart for the most basic cases?

I guess we should, you're right.

Reply:
> Do we need to expose full configuration to customers?

I thought it would be useful to do this since there are multiple ways to configure the receiver and the benefit of exposing it this way could be that we wouldn't need to release new versions for different cloudwatch configuration use cases that multiple customers might have, as shown below:

1st use case (using autodiscover)
```
awscloudwatch:
  region: us-west-1
  logs:
    poll_interval: 1m
    groups:
      autodiscover:
        limit: 100
        prefix: /aws/eks/
        streams:
          prefixes: [kube-api-controller]
```

2nd use case: (using named)

```
awscloudwatch:
  region: us-west-1
  logs:
    poll_interval: 5m
    groups:
      named:
        /aws/eks/dev-0/cluster: 
          names: [kube-apiserver-ea9c831555adca1815ae04b87661klasdj]
```